src/main/resources/com/google/api/codegen/java/settings.snip

```diff
@@ -345,7 +345,9 @@
           }
           @@Override
           public Iterable<{@desc.resourceTypeName}> extractResources({@desc.responseTypeName} payload) {
-            return payload.{@desc.resourcesFieldGetFunction}();
+            @let resourcesGetFunction = functionChain(desc.resourcesFieldGetFunction)
+              return payload.{@resourcesGetFunction};
+            @end
```

Comment:
Why are you introducing usage of `@let` here?

Reply:
Removed `@let` and just put the function call in that `return payload...` line.